tests/conftest.py

```diff
@@ -3,7 +3,7 @@
 import pytest
 import xarray as xr
 
-from tests._common import write_roocs_cfg
+from tests._common import *
```

Comment:
For `*` imports, having a predefined `__all__` in `tests._common` will prevent us from importing things like `jinja2` here. I can add this if we want.

Reply:
yes please